src/layout/header/Logo.tsx

```diff
@@ -1,13 +1,33 @@
-import { Box, Button, Typography } from '@mui/material';
+import { Box, Button, Theme, Typography, useMediaQuery } from '@mui/material';
+import { useTranslation } from 'react-i18next';
 import { Link as RouterLink } from 'react-router-dom';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
-export const Logo = () => (
-  <Box sx={{ gridArea: 'logo' }}>
-    <Button data-testid="logo" component={RouterLink} to={UrlPathTemplate.Home}>
-      <Typography variant="h5" component="span" sx={{ color: 'white', fontWeight: 900, fontSize: '3rem' }}>
-        NVA
-      </Typography>
-    </Button>
-  </Box>
-);
+export const Logo = () => {
+  const { t } = useTranslation();
+  const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('md'));
+  return (
+    <Box sx={{ gridArea: 'logo' }}>
+      <Button data-testid="logo" component={RouterLink} to={UrlPathTemplate.Home}>
```

Comment:
`Box` trengs ikke her, da den bare har en knapp som kan styre sin plassering selv
```suggestion
      <Button sx={{ gridArea: 'logo' }} data-testid="logo" component={RouterLink} to={UrlPathTemplate.Home}>
```